lib/ansiblelint/utils.py

```diff
@@ -53,6 +59,7 @@
     from ansible.parsing.mod_args import ModuleArgsParser
     from ansible.parsing.yaml.constructor import AnsibleConstructor
     from ansible.parsing.yaml.loader import AnsibleLoader
+    from ansible.parsing.yaml.objects import AnsibleSequence
     from ansible.errors import AnsibleParserError
     ANSIBLE_VERSION = 2
```

Comment:
Wow, it looks like this whole try/except needs to be cleaned up because we don't support Ansible 1

Reply:
But that's out of scope right now. My concern is that if someone is running it with Ansible 1 for whatever reason, they may hit `NameError`